kedro/templates/project/prompts.yml

```diff
@@ -1,3 +1,22 @@
+add_ons:
+  title: "Project Add-Ons"
+  text: |
+    Here you can select which add-ons you'd like to include.
+    Don't worry if you change your mind you can always add/remove these later.
+    To read more about these add-ons and what they do visit: kedro.org/{insert-documentation}
+
+    Add-Ons
+    1) Linting : Provides a basic linting set up with Flake8, Black and isort
```

Comment:
Do we want to do `ruff` instead? cc: @noklam

Reply:
Yes please 🙏🏽